azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/VirtualMachineScaleSetExtension.java

```diff
@@ -7,7 +7,6 @@
 package com.microsoft.azure.management.compute;
 
 import com.microsoft.azure.management.apigeneration.Fluent;
-import com.microsoft.azure.management.compute.implementation.VirtualMachineScaleSetExtensionInner;
```

Comment:
Needed this import once the inner is moved to implementation namespace (see below comment)